Update Dynamo sample graphs

dnenov opened the pull request:
<!--
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.
-->

### Purpose

An update to Dynamo sample graphs for the following categories:
- Basics (3 graphs)
- Core (9 graphs)
- Geometry (4 graphs)
- ImportExport (5 graphs)

Contains 21 updated graphs in total.

### Issues
There seem to be an issue for the `Data.OpenXMLExportExcel` node, potentially connected to .net API change.

![image](https://github.com/DynamoDS/Dynamo/assets/5354594/be765ac4-2863-4be0-8cba-890e6843f60a)

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [x] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB

### Release Notes

- update all dynamo-related sample graphs


### Reviewers

@QilongTang 
@Amoursol 

### FYIs

@dariaivanciucova

QilongTang commented:
I think the EngOps build failure might be the same we saw on master-15

QilongTang commented:
hi @dnenov Did you add new nodes to the samples? Please update the following tests because they are failing both on our CI and locally for me.
<img width="1384" alt="image" src="https://github.com/DynamoDS/Dynamo/assets/3942418/6e69d6eb-6519-401b-95c3-ec1c6d584ab9">

dnenov commented:
> hi @dnenov Did you add new nodes to the samples? Please update the following tests because they are failing both on our CI and locally for me.

Oh, yes, we did update a lot of nodes in the graphs, as this was the task 
- we have remove old nodes that don't exist anymore
- we have renamed Code Block nodes for clarity
- we have grouped nodes to enhance readability 

I have updated all the failing tests from the list. All tests passing locally.
